bokehjs/src/coffee/models/glyphs/image.ts

```diff
@@ -48,6 +52,55 @@ export class ImageView extends XYGlyphView {
     }
   }
 
+  _index_data(): RBush {
+    const points = [];
+    for (let i = 0, end = this._x.length; i < end; i++) {
+      const [l, r, t, b] = this._lrtb(i);
+      if (isNaN(l+r+t+b) || !isFinite(l+r+t+b)) {
+        continue;
+      }
+      points.push({minX: l, minY: b, maxX: r, maxY: t, i});
+    }
+    return new RBush(points);
+  }
+
+  _lrtb(i: any) {
+    const l = this._x[i]
+    const r = l + this._dw[i]
+    const b = this._y[i]
+    const t = b + this._dh[i]
```

Comment:
Maybe switch the lines here so that they are in lrtb order to avoid cognitive dissonance. :-)

Reply:
Sure.

Reply:
Actually, I undid that commit as ``t`` is defined using ``b`...

Reply:
Ah, good point!